### ISSUE TITLE
Luamin makes my script larger? 

### ISSUE THREAD

**Issue by teckel12** [mathiasbynens/luamin#47]
Oddly enough, luamin made my script larger (from 20k to 24k). Also, execution failed as it seems lines over 256 characters are truncated (at least in my Lua parser, OpenTX).

**Comment by teckel12** [comment]
Any chance of creating an option to keep the lines shorter than 256 characters so I can use it with the OpenTX Lua Script?

**Comment by teckel12** [comment]
Tried to duplicate but could not.  Luamin reduced the file size by 39%.

I'm using Lua on an OpenTX radio control transmitter which can't have lines longer than 255 characters.  I manually cropped down the lines to be less than 255 characters long and it compiled and ran without a hitch.

Unfortunately, minimizing the source with Luamin didn't resolve my problem.  My issue was the limited RAM available on the transmitter.  Minimized code didn't change the memory used (the memory usage peek happens at the time of compile).  Instead, my solution was to  distribute an already compiled Lua script, which greatly reduces the peek memory usage.

Thanks, I'll close this as I must have been mistaken when I opened it.
